addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -211,22 +208,20 @@ private void updateFanartUrl(String imagePath) {
         /*
```

Comment:
It was there in the initial commit, so if @pail23 has no objections I would like to remove it.